core.md

```diff
@@ -0,0 +1,111 @@
+# Governance of the NeoForged project
+
+The NeoForged project is split between [github](https://github.com/NeoForged) and [discord](https://discord.gg/UvedJ9m). It is a community of volunteers, maintaining a free and open source ecosystem of modifications to the game Minecraft.
+
+### What this document is
+
+This document is:
+ * A non-exhaustive list of the working principles of the project.
+ * Intended to help direct the project in times of uncertainty.
+
+This document is built on the fundamental principle of "people generally want to help, so we should not get in the way of it".
+
+The core ethic is "people over processes", so this document is intentionally light on processes - situations that come up are rarely one-size-fits-all, and should be treated on a case by case basis, with the best process to fit the person.
+
+### What this document is not
+
+This document is not:
+ * Intended to enforce rules on all members.
+ * Intended to catch all possibilities.
+ * Intended to prevent "rules lawyering".
+
+The code of conduct of the project is taken to be the content of [the Contributor Covenant](https://www.contributor-covenant.org/version/2/1/code_of_conduct/), so this document is not the code of conduct.
+
+Conflict resolution is the responsibility of the Moderation and Steering Council teams. They may wish to append a section to this document, but currently, this document is not conflict resolution.
+
+## Definitions
+
+Following are some core definitions that will be referred to in the later sections of the document.
+
+### The Project
+
+"The Project" refers to The NeoForged project, with a presence on both GitHub and Discord, as noted at the start of this document. It does not refer to any specific Repository or subproject.
+
+### Subprojects
+
+Subprojects are large tasks that the Project itself is undergoing.
+
+These may be:
+
+* Repositories under the Project's GitHub
+* Large, multi-repository tasks that are being undertaken (ie. updating the Modloader to a new version of the Game)
+* Smaller, more targetted tasks that are being undertaken (ie. refactoring the Modloader to remove or replace or rework a feature)
+
+### Teams
+
+The Project is split into four teams.
+
+A team is a group of people with similar goals, and equal weighting in most decision making - with some exceptions, which will be explored later.
+
+You may [apply to join some teams](https://tinyurl.com/ForgeApplications), but some require appointment from within an existing team, or further voting after application.
+
+#### Steering Council
```

Comment:
This section should mention that the powers of the council are not all-encompassing and that decisions on different matters shall be first ran by the team responsible and may only be vetoed by the council under exceptional reasoned circumstances (“drama”, draws).